src/gdb_main.c

```diff
@@ -408,14 +422,12 @@ static void exec_q_rcmd(const char *packet, const size_t length)
 
 	const int result = command_process(cur_target, data);
 	if (result < 0)
-		gdb_putpacketz("");
+		gdb_put_packet_empty();
 	else if (result == 0)
-		gdb_putpacketz("OK");
+		gdb_put_packet_ok();
 	else {
 		const char *const response = "Failed\n";
```

Comment:
Given how this gets used, it probably makes sense to make this a `static const char []` and then use `ARRAY_LENGTH()` in the put packet call (trimming 1 from the result to account for the '\0' implied at the end of the string).

Reply:
The idea being removing the need for strlen?

Reply:
yep, exactly - it's a static string, so let's get as close to a std::string_view as we can in C so we're not wasting cycles on information we already have.

Reply:
I would wager it gets optimized out, but using ARRAY_LENGTH doesn't hurt